cpp/src/cider/exec/nextgen/context/CodegenContext.cpp

```diff
@@ -386,5 +393,17 @@ void bitBufferAnd(jitlib::JITValuePointer& output,
           .params_vector = {output.get(), a.get(), b.get(), bit_num.get()}});
 }
 
+void convertByteBoolToBit(jitlib::JITValuePointer& byte,
+                          jitlib::JITValuePointer& bit,
+                          jitlib::JITValuePointer& len) {
+  CHECK((isaTypedJITValue<JITTypeTag::POINTER, JITTypeTag::INT8>(byte)));
```

Comment:
if CHECK failed, FATAL message log will print, and application will abort.
Is that your are expected?

Reply:
Yes, I expect all potential errors of the code generation will expose explicitly in tests.